<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEmployeeDialog</class>
 <widget class="QDialog" name="AddEmployeeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить Сотрудника</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>Добавить Сотрудника</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSurname">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditSurname"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPatronymic">
       <property name="text">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditPatronymic"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPassport">
       <property name="text">
        <string>Паспорт(серия, номер)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditPassport"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelJobTitle">
       <property name="text">
        <string>Должность</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditJobTitle"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelMedicalPolicy">
       <property name="text">
        <string>Мед.Полис</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditMedicalPolicy"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelInsurancePolicy">
       <property name="text">
        <string>Страховой Полис</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditInsurancePolicy"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelVacation">
       <property name="text">
        <string>Отпуск</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QDateEdit" name="dateEditVacationStart">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTo">
         <property name="text">
          <string>—</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEditVacationEnd">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
